### DISCUSSION TITLE
[DNS] Not found - DNS TXT verification record not found.

### DISCUSSION THREAD

**Discussion by overplex-eth** [WalletConnect/walletconnect-monorepo#4363] [Developer Support]
I keep getting this error. I checked with https://mxtoolbox.com and the txt on the DNS is found. Maybe i've wrongly put the DNS Configuration. 

Here's the picture:
![Screenshot 2024-03-25 180943](https://github.com/WalletConnect/walletconnect-monorepo/assets/112352174/5293b8c0-e697-4b17-b1bc-7d5e8e8bdd78)

Thanks!

**Comment by Cali93** [comment]
Hey @overplex-eth,

Sorry for the very late reply. Did you figure out the issue ? 
Could you share your `projectId` and `domain` ?
